contracts/integrations/test/framework/actors/base.ts

```diff
@@ -32,7 +33,7 @@ export class Actor {
 
     constructor(config: ActorConfig) {
         Actor.count++;
-        this.address = config.deployment.accounts[Actor.count];
+        this.address = config.deployment.accounts[Actor.count % config.deployment.accounts.length];
```

Comment:
don't we kind of want it to throw if the index overflows? otherwise we might unwittingly use the same address somewhere

Reply:
After discussing offline, we will use another solution to this problem and add a better error message.